tests/pom.xml

```diff
@@ -107,6 +129,12 @@
             <artifactId>jersey-apache-connector</artifactId>
             <version>${jersey-version}</version>
         </dependency>
+        <!--  Gson: Java to Json conversion -->
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.8.8</version>
+        </dependency>
```

Comment:
Can you remove this as its no longer needed as per Sai's comment?

Reply:
done https://github.com/factset/analyticsapi-engines-java-sdk/pull/63/commits/c61a100671f11f92d947d852c8d9c8274f7286d4